common/test/keyboards/build.sh

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../../resources/build/build-utils.sh"
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
```

Comment:
I presume KEYMAN_ROOT is set up an all OSes that will be using this build test script.

Reply:
`KEYMAN_ROOT` is declared by `build-utils.sh` -- that's why this is a standard boilerplate for all our scripts (some older ones haven't been updated yet). It works cross-platform and deals with the irregularities. Once we got it right, the boilerplate has avoided so many hassles.